advanced-vector-search/flow-index.yml

```diff
@@ -6,9 +6,8 @@ pods:
     uses: _forward
```

Comment:
We still need this forward? I remember we had it to make it consistent with the other examples we had but is still that the case?

Reply:
it is not needed, we can remove